Reflections/ReflectionWeek3.md

```diff
@@ -0,0 +1,15 @@
+1. For this week I have accomplished most of everything that I wanted to do:
+- I fixed my logo, added a blacked gradient background, and changed the colors of my buttons so I could have a consistent theme.
+- I also implemented the spinning wheel, so know when the user finishes the filters section they have a colorful wheel that will be displayed before the restaurant is chosen.
+- I know I can't use it just yet but I also created a System Usability Survey for my app to get feedback from friends and family members that I have told about my project.
+
+2. The biggest challenge that I faced this week was the implementation of the spinning wheel. The links to the youtube tutorials that I have previously listed, were older versions of the thunkable website and couldn't be used. I basically went to the Thunkable forums to try and figure out what was different.(https://community.thunkable.com/t/image-rotation-not-available-in-thunkable-x/1061768/4) Turns out that instead of allowing images to rotate, they made them into sprites on a canvas. This made it easier for anyone using Thunkable to create a mobile game.
+
+3. Now that I have a big majority of my app finished I am not sure what to do now. I did talk with one of my friends over the weekend and we discussed filterting the restaurants by breakfast, lunch, and dinner. However in the Yelp API documentation those don't show up in the categories.
+- I did some research and instead of me setting up switches for different filters, I am thinking about adding in a text input box, for the user to input filters. The main problem with this is spelling and whether or not the search filter is built into the Yelp API. This probably would only take me about an hour or two as long as I can understand how the Yelp API does categories.
+- I do have to try and clean up my app a little more and Imma suggested that I should a add a really cool UI design for my app. I have actually thought about animating the logo of the app by having the fork and knife move up and down and having the wheel spin. I don't know my about animation in thunkable but I will look more into it. This isn't something I am 100 percent sure on because it might require me to separate the logo into 3 gifs and I'm not sure how they will fit on the screen. If I watch enough videos I am sure I could accomplish this in a few hours by changing the y-axis of the fork and knife and just rotating the logo.

```

Comment:
I found this thunkable blog where people are talking about options for animating images. 
https://community.thunkable.com/t/whats-the-best-way-to-display-animated-gifs-solved/274715/10